backend/amd64/emit.mlp

```diff
@@ -790,6 +803,21 @@ let emit_instr fallthrough i =
         def_label label;
         I.lea (mem64 NONE 8 R15) (res i 0)
       end
+  | Lop(Ialloc { bytes = n; dbginfo=_; mode = Alloc_local }) ->
```

Comment:
Does a local `Ialloc` need a frame table entry?

Reply:
No, it never enters the garbage collector. (It may call caml_local_realloc if it needs more stack space, but that never GC's)

Reply:
ok, thanks for clarifying. Everything transitively reachable from local stack must be on the local stack,right? Is it allowed to point outside the ocaml heap (e.g., to some C struct)?

Reply:
No, the local stack can point to GC heap values. The GC heap may never point to the local stack, though. (This means that the GC must scan the local stack for roots as well, since a value may be live only through the local stack)

Reply:
got it.. I think: Local alloc doesn't call the GC, so it doesn't need a frame descriptor, but local roots can be marked as live in other (normal) frame descriptors.